cmd/logos/logo-template.txt

```diff
@@ -0,0 +1,5 @@
+package main
+
+import "image/color"
+
+var logoRGBA = []color.RGBA{ {{.LogoRGBA}} }
```

Comment:
Now that TinyGo has support for `embed` I wonder if we should try switching to that?